db/schema.rb

```diff
@@ -153,6 +168,29 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "name"
+    t.string   "self_care_strategy"
+    t.boolean  "daily_reminder_email"
+  end
+
+  create_table "strategy_email_reminders", force: :cascade do |t|
```

Comment:
i actually deleted dose tables, m shocked the migration files r missing while i have them here i will pull frm master, create just the needed tables and do a fresh pr